### PR TITLE
adding mp3 scraper

### PR DESCRIPTION
**Pull request by Jkellys**
freesound mp3 scraper

### CODE REVIEW

**Comment by julesyoungberg** [comment]
Thanks for getting this going! 

I think the next steps here are to take some configuration arguments (e.g. a path to a JSON file that contains a list of instruments to collect samples on, the number of samples to collect for each instrument). The script should also save the samples into directories that match the instrument so we can easily use them for ML training.

@chrispetrone thoughts?

**Comment by squisho** [comment]
If freesound offers .wav we should use that, otherwise we have to build a script to convert the mp3s into wavs. 

so far, most of the models I've seen convert wavs into Spectragrams and MFCCs or other image-forms, and then use an image-based model to classify them.

**Comment by Jkellys** [comment]
ok, I will look into the JSON file and matching directories. From what I have seen on Freesound it only allows mp3 so I think we are going to need to make our own script or find a library to convert them to mp3.

**Comment by squisho** [comment]
@Jkellys from what I can see it looks like we would have to configure OAuth and switch to HTTP requests to download full-res soundfiles.

File: freesound_scraper.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+			
+			# off for testing on the ferry
+			get_sounds(i)
+			os.chdir("/Users/jonathankelly/Desktop/csc_475/project/instaments/")
+		except OSError as error:  
+			print(error)  
+
+
+
+home_path = "/Users/jonathankelly/Desktop/csc_475/project/instaments"
```
Comment (julesyoungberg) [inline freesound_scraper.py:115]: I would resolve relative paths here, maybe something like:
```python
os.path.join(os.path.dirname(__file__), './instruments')
```

**Review by julesyoungberg — APPROVED** [review]
This is awesome man! One note on paths but otherwise looks great!

**Pull request merged**